.eslintrc.yml

```diff
@@ -0,0 +1,175 @@
+rules:
```

Comment:
Probo rules needed to be updated... https://github.com/ProboCI/eslint-config-probo/pull/1